Composer/packages/client/src/components/ProjectTree/ExpandableNode.tsx

```diff
@@ -64,25 +64,20 @@ export const ExpandableNode = ({
   }
 
   return (
-    <details
+    <ul
       ref={detailsRef}
       aria-expanded={isExpanded}
-      css={nodeStyle}
+      css={listStyle}
       data-testid="dialog"
-      open={isExpanded}
       role="tree"
+      onClick={handleClick}
+      onKeyDown={handleKey}
     >
       {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/no-noninteractive-tabindex */}
-      <summary
-        css={summaryStyle(depth, isActive, isExpanded)}
-        data-testid={'summaryTag'}
-        tabIndex={0}
-        onClick={handleClick}
-        onKeyUp={handleKey}
-      >
+      <li css={listItemStyle(isActive, isExpanded)} data-testid={'summaryTag'}>
         {summary}
```

Comment:
Can we safely assume that any text in {summary} jsx component is localized?

Reply:
Mostly they're going to be user-input names of bots and dialogs, but in the few cases where someone's making a tree for another purpose, that will have been localized already in the source component.